services/web-app/components/divider/divider.module.scss

```
@@ -0,0 +1,19 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+
+.divider {
+  padding-top: spacing.$spacing-04;
+  border-top: 1px solid theme.$border-strong;
+  margin-bottom: calc(spacing.$spacing-09 + spacing.$spacing-03);
```

```suggestion
  padding: spacing.$spacing-04 spacing.$spacing-05 0 0;
  border-top: 1px solid theme.$border-strong;
  margin-right: -(spacing.$spacing-05);
  margin-bottom: calc(spacing.$spacing-09 + spacing.$spacing-03);
```

@francinelucca with the eagle eyes! you're correct its 1rem off on the right, this should fix. 

It wouldn't let me do shorthand for margin with the negative value in the middle 🤷‍♀️